src/tz.cpp

```diff
@@ -118,6 +118,9 @@
 // the current time zone. On Win32 windows.h provides a means to do it.
 // gcc/mingw supports unistd.h on Win32 but MSVC does not.
 
+#ifdef __ANDROID__
+#  define INSTALL .
```

Comment:
Previoysly I did the same thing in Kodi but speaking of the upcoming version tag - #752 brings CMake option to opt out of installation so this hunk becomes redundant